src/main/app/src/app/informatie-objecten/informatie-objecten.service.ts

```diff
@@ -42,8 +43,8 @@ export class InformatieObjectenService {
         );
     }
 
-    listEnkelvoudigInformatieobjectenVoorZaak(uuid: string): Observable<EnkelvoudigInformatieobject[]> {
-        return this.http.get<EnkelvoudigInformatieobject[]>(`${this.basepath}/zaak/${uuid}`).pipe(
+    listEnkelvoudigInformatieobjecten(zoekParameters: EnkelvoudigInformatieObjectZoekParameters): Observable<EnkelvoudigInformatieobject[]> {
+        return this.http.post<EnkelvoudigInformatieobject[]>(`${this.basepath}/informatieobjectenList`, zoekParameters).pipe(
```

Comment:
Je hebt hier een post van gemaakt, ik neem aan omdat je een body meegeeft. Maar in rest is het daardoor nog geen post, er wordt niet elke keer dat je 'm aanroept iets opgeslagen.

Reply:
put dan maar, aangepast